app/controllers/user_controller.js

```diff
@@ -19,14 +19,73 @@ exports.signUp = (req, res, next) => {
     });
 };
 
-exports.signIn = function(req, res, next) {
+exports.signIn = (req, res, next) => {
   const { email, password } = req.body;
   return User.findOne({ where: { email } }).then(user => {
     if (!user) next(error.unAuthorizedError('Invalid credentials'));
```

Comment:
its the same error both times, just do `if (!user || !user.validatePassword(password)`